### PR TITLE
docs: prose overview for the interaction diagram for Zoe and Zcf

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
Includes security claims about the split, as well as description of
how Zoe and Zcf communicate while making new instances of contracts.

https://github.com/Agoric/agoric-sdk/pull/1288
https://github.com/Agoric/agoric-sdk/issues/553

### CODE REVIEW

File: packages/zoe/docs/zoe-zcf.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Zoe and ZCF
+
+At the current time, Zoe is split into an "upper" part, which is the facet that the contract communicates with, and the "middle", which contains all the common pieces shared across all contracts. The middle has the InviteIssuer, manages the purses for all contract instances, and is the central point of contact for installing and instantiating contracts. Each time a contract is instantiated with makeInstance(), a new vat is created, and the code for ZCF (the Zoe Contract Facet) is executed, immediately running the code for the contract itself.
```
Comment (katelynsills) [inline packages/zoe/docs/zoe-zcf.md]: Let's not use upper and middle

File: packages/zoe/docs/zoe-zcf.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+ * [Plan for current split](https://github.com/Agoric/agoric-sdk/issues/553)
+ * [PR for split](https://github.com/Agoric/agoric-sdk/pull/1288)
+ * ![Zoe-Zcf interaction diagram](./zoe-zcf.png)
+
+### interactions between Zoe and Zcf
+
+Zoe has a single InviteMint, which it uses to mint Invitations which verify the authenticity of contracts and seats. Zoe freely shares the corresponding Issuer, so everyone can recognize valid Invitatations.
+
+Zoe is created with SwingSet. Any user can call `E(zoe).install(bundle)` to create a new verified installation of a contract. From that point, anyone can validate  the source code, or get assurances from Zoe that a particular instance of that contract is running that code.
```
Comment (katelynsills) [inline packages/zoe/docs/zoe-zcf.md]:
```suggestion
Zoe is created by CosmicSwingset. Any user can call `E(zoe).install(bundle)` to create a new installation of a contract. From that point, anyone can use the installation to view the code and to get assurances from Zoe that a particular instance of that contract is running that code.
```

**Review by katelynsills — APPROVED** [review]
LGTM!

File: packages/zoe/docs/zoe-zcf.md (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/zoe/docs/zoe-zcf.md]: okay. I didn't find consensus terminology for zoe main, so I made something up.

File: packages/zoe/docs/zoe-zcf.md (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/zoe/docs/zoe-zcf.md]: done.

**Pull request merged**